app/controllers/credit_cards_controller.rb

```diff
@@ -6,6 +6,7 @@ def index
   end
 
   def show
+    @credit_card = CreditCard.find(params[:id])
```

Comment:
I believe I already have this line in the private portion of code. Can be removed